src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+.header {
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family is better moved to the "html" tag and you won't need to place it anywhere else